node/device/core/lib/blob_upload/blob_uploader.d.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Stream } from 'stream';
+
+import BlobUploadClient = require('./blob_upload_client');
+
+declare class BlobUploader implements BlobUploadClient.BlobUploader {
+    constructor(storageApi?: BlobUploader.StorageApi);
+
+    uploadToBlob(uploadParams: BlobUploadClient.UploadParams, stream: Stream, streamLength: number, done: (err: Error, body?: any, result?: { statusCode: number, body: string }) => void): void;
+}
+
+declare namespace BlobUploader {
```

Comment:
They do, but the class in their typings has a lot of many other non-optional functions that our module doesn't use. This is minimal functionality that our SDK requires and needs to be replaced for testing/re-implementation.